docs/UserGuide.md

```diff
@@ -477,7 +480,9 @@ Therefore, edit the data file only if you are confident that you can update it c
 ## FAQ
 
 **Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous StaffConnect home folder.
+**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous StaffConnect home folder.<br>
```

Comment:
i think `overwrite the empty data file` is inaccurate, perhaps `overwrite the generated data file with the file...` (removed the `it creates` because `generated` implies creation)

Maybe `contains the data of your previous StaffConnect home folder` -> `contains the data of the StaffConnect on your current device` is better? maybe it's just me, i personally do not understand what `home folder` is referring to 😅

Reply:
Wait this statement is from the default address book I only added the line break 🤣

Reply:
Fixed, and added slightly more content